app/uk/gov/hmrc/transitmovementsrouter/controllers/MessagesController.scala

```diff
@@ -104,10 +107,13 @@ class MessagesController @Inject() (
       implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
       (for {
         _ <- parseAndLogUpscanResponse(request.body)
-        //TODO: Get the object store uri location and extract the message type, remove below hardcoded value and then pass actual value with below method
-        //TODO: Get the object store uri from object store remove below hardcoded value and then pass that too with below method
+        //TODO: Get the object store uri from object store remove below hardcoded value and then pass that with below method
+        objectStoreResourceLocation <- extractObjectStoreResourceLocation(ObjectStoreURI("common-transit-convention-traders/movements/abc.xml"))
```

Comment:
What does object store return when you copy a file across? Will this be necessary?

Best to talk with @olakunlekuye2 about this as he will know what he's returning.

Reply:
It returns a ObjectSummaryWithMd5 object of which you can access the location (file) amongst other things i.e objectStoreResourceLocation.location.

Reply:
Thanks both. I know what needs to be done here. it is going to be ``` objectSummaryWithMd5.location.asUri ```. this will return uri by including and starting with ``` common-transit-convention-traders ``` .  But the thing is that Ola changes are still not merged with main and I did my changes before he raised pull request for his ticket. I will take the Ola branch and do the all changes over here by assuming that his changes will be merged first then mine.